content/es.kindly

```diff
@@ -0,0 +1,22 @@
+The KINDLY License
+Copyright (c) <year> <author>
+
+Tienes la autorización para utilizar este software, leer su código, modificarlo y distribuirlo bajo las siguientes condiciones:
```

Comment:
Thanks! :sparkler:

Please format it to have max 80 columns, like in `license.kindly`. Also, please move the `content.kindly` file into `content/en.kindly`.